cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -65,3 +68,7 @@ alpha:
 
 metrics:
   enabled: false
+
+nodeSelector: {}
+tolerations: {}
+affinity: {}
```

Comment:
nitpick: can we move these up below the `image` configuration values so that it is more clear they are associated with the core Crossplane deployment?

Reply:
I have moved these settings.   To me as a consumer of the chart it would make more sense for all the crossplane specific settings to live under `crossplane:` 
Ex.
```
crossplane:
  nodeSelector: {}
  tolerations: {}
  affinity: {}
```
oftentimes there is a `global:` section which takes precedence.
that is the typical pattern and makes it clear what is consuming which value.

Reply:
agreed, this is just a legacy of the fact that the RBAC manager didn't exist initially. Also, can you make sure to update those two docs when you get a chance?